recipe/meta.yaml

```diff
@@ -26,15 +27,20 @@ requirements:
     - {{ compiler('cuda') }}
     - make
     - cmake
+    - mlir =14
   host:
     - python
+    - pybind11
     - pip
-    - llvmdev =11
+    - llvmdev =14
     - libzlib
+    - mlir =14
   run:
     - python
     - filelock
     - pytorch =*=cuda*
+    - libllvm14
```

Comment:
Shouldn't this run export happen automatically

Reply:
No, if that’s not added the test falls with a missing library.

Reply:
I'm always confused as to the best practices for using LLVM.

For pyside2 we used the package `llvmdev` and added the pinnings to the config. To my knowledge, this is the preferred way of doing it.

https://github.com/conda-forge/pyside2-feedstock/commit/b0a41f3db265a7654751b9a6d992dd5c43b3f93c

Reply:
Fails with `ImportError: libLLVM-14.so: cannot open shared object file: No such file or directory` if not added manually.

Reply:
maybe the right package is `llvm` and not `llvmdev`

Reply:
I'm not sure how you feel about these changes. they seem to work locally, but I cleaned up the history, so maybe the tests might fail:
https://github.com/conda-forge/triton-feedstock/pull/4/commits/59f1b1b741a19e7b01b1c195a4e1b016f0af88e2